packages/flutter_markdown/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.7.3+2
+
+* Resolved an issue where code blocks in markdown were not highlighted during selection.
```

Comment:
"Resolves"

Please see the CHANGELOG style guide linked from the PR checklist. (Please also update the checklist to reflect the current state of the PR.)

Reply:
Done